collection/timothy.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "timothy",
+    "meaning": "Greek for \"honouring God\"",
+    "aliases": [
+        "tim",
+        "timmy"
+    ],
+    "translations": {
+        "ell": "Τιμόθεος"
```

Comment:
All translations should be lower case.
Please change it so I could merge your PR.
Thanks!